alfresco-transform-libreoffice/alfresco-transform-libreoffice-boot/src/main/java/org/alfresco/transformer/Application.java

```diff
@@ -60,6 +59,9 @@
         return registry -> registry.config().commonTags("containerName", containerName);
     }
 
+    // To run the LibreOffice T-Engine from the command line on a Mac, you generally need to
```

Comment:
Minor: could also go into project README (if not already) - ie.  typical minimum options to run each core T-Engine (on Mac, *nix or Windows).

Reply:
It could, but that would require quite a bit of work to add a section like that. I would suggest that would need to be part of a separate Epic.

Reply:
Ok, thanks.